dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task230420AlterVarcharLengthOfLockedByColTest.java

```diff
@@ -0,0 +1,70 @@
+package com.dotmarketing.startup.runonce;
+
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import org.junit.Test;
+
+import java.sql.SQLException;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class Task230420AlterVarcharLengthOfLockedByColTest {
+    // Change the length to recreate the scenario
+    private void setMinLengthBeforeTask (String tblName) throws SQLException {
+        String query = "";
+        if (DbConnectionFactory.getDBType().equals("PostgreSQL")){
```

Comment:
use DbType.POSTGRESQL == DbType.getDbType(DbConnectionFactory.getDBType())

Reply:
You have to close the connection on a finally

Reply:
Sure, I'll change it

Reply:
Done
<img width="679" alt="image" src="https://user-images.githubusercontent.com/127987858/233796453-c46a1148-7f91-4ed5-9bc6-c5893bfc834e.png">